### PR TITLE
modify confluent doc based on verification

### PR DESCRIPTION
**Pull request by xuweibj**
Task: https://github.com/xcat2/xcat2-task-management/issues/198

After modified: 
http://10.3.17.21/install/html/advanced/confluent/index.html

### CODE REVIEW

File: docs/source/advanced/confluent/client/confluent_client.rst [hunk 1]
```diff
@@ -40,13 +40,19 @@ If you want to use the xCAT Keys, you can simple copy them into ``/etc/confluent
     cp /etc/xcat/cert/server-key.pem /etc/confluent/privkey.pem
     cp /etc/xcat/cert/server-cert.pem /etc/confluent/srvcert.pem 
 
+The user and password may alternatively be provided via environment variables: ::
+
+    CONFLUENT_USER=vhu
```
[the remark this answers is not in the document]
Comment (whowutwut) [reply docs/source/advanced/confluent/client/confluent_client.rst]: Why do  we need this exports? What is the error that comes out if we do not define it?

[the remark this answers is not in the document]
Comment (whowutwut) [reply docs/source/advanced/confluent/client/confluent_client.rst]: Oh, maybe part of that TODO below... on line 50.

[the remark this answers is not in the document]
Comment (xuweibj) [reply docs/source/advanced/confluent/client/confluent_client.rst]: @whowutwut yes, I just modified "TODO" to define env variables depending on confluent doc.

**Comment by zet809** [comment]
Looks good to me, let me merge it. Thx @xuweibj

**Pull request merged**